src/Renderer/ThreeExtented/GlobeControls.js

```diff
@@ -471,41 +469,39 @@ function GlobeControls(camera, domElement, engine) {
     var update = function() {
 
         if (state === CONTROL_STATE.MOVE_GLOBE) {
-            offset.copy(snapShotCamera.position);
-        } else {
-            // get camera position
-            offset.copy(this.camera.position);
-            // transform to local globeTarget
-            offset.applyMatrix4( globeTarget.matrixWorldInverse );
-        }
-
-        offGT.copy(globeTarget.position);
 
-        if (state === CONTROL_STATE.MOVE_GLOBE) {
+            movingGlobeTarget.copy(this.getCenter()).applyQuaternion(quatGlobe);
+            this.camera.position.copy(snapShotCamera.position).applyQuaternion(quatGlobe);
+            this.camera.up.copy(movingGlobeTarget.clone().normalize());
```

Comment:
Again: adding a single line of comment explaining what you're doing here would be useful.
Examples: 
```js
// Rotate globe then update camera to keep camera's center aligned
// Pan camera along globe's normal
```